components/header/LayoutSwitch.tsx

```
@@ -0,0 +1,57 @@
+        <div className="absolute right-0 mt-2 w-56 bg-white dark:bg-gray-900 border rounded shadow-lg z-50">
+          {LAYOUTS.map((layout) => (
+            <button
+              key={layout.key}
+              className={`flex items-center w-full px-4 py-2 hover:bg-gray-100 dark:hover:bg-gray-800 ${
+                value === layout.key ? "bg-gray-100 dark:bg-gray-800" : ""
+              }`}
+              onClick={() => {
+                onChange(layout.key);
+                setOpen(false);
+              }}
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)